### PR TITLE
update for SDK 6.0

### PR DESCRIPTION
**Pull request by eli-darkly**
We'll merge this to master (and change the dependency from `6.0.0-rc.1` to `6.*`) once we've done the GA release.

### CODE REVIEW

File: HelloDotNet/Hello.cs [hunk 1]
```diff
@@ -26,16 +25,11 @@ static void Main(string[] args)
                 Environment.Exit(1);
             }
 
-            LogManager.Adapter = new ConsoleAdapter(LogLevel.Info);
-            // ConsoleAdapter is a simple Common.Logging implementation that's provided by LaunchDarkly;
-            // Common.Logging has an equivalent class, but not on every platform
```
Comment (eli-darkly) [inline HelloDotNet/Hello.cs:31]: We're no longer using Common.Logging, and the SDK now provides its own default logging behavior which is equivalent to this (output to the console, at Info level).

File: HelloDotNet/Hello.cs [hunk 2]
```diff
@@ -26,16 +25,11 @@ static void Main(string[] args)
...
-            LogManager.Adapter = new ConsoleAdapter(LogLevel.Info);
-            // ConsoleAdapter is a simple Common.Logging implementation that's provided by LaunchDarkly;
-            // Common.Logging has an equivalent class, but not on every platform
+            var ldConfig = Configuration.Default(SdkKey);
 
-            Configuration ldConfig = LaunchDarkly.Client.Configuration
-                .Default(SdkKey);
+            var client = new LdClient(ldConfig);
 
-            LdClient client = new LdClient(ldConfig);
```
Comment (eli-darkly) [inline HelloDotNet/Hello.cs:36]: Here I've just changed explicit type declarations to `var`— nothing to do with the new SDK, just good C# practice.

File: HelloDotNet/Hello.cs [hunk 3]
```diff
@@ -26,16 +25,11 @@ static void Main(string[] args)
...
-
-            if (client.Initialized())
+            if (client.Initialized)
```
Comment (eli-darkly) [inline HelloDotNet/Hello.cs:32]: `Initialized` is a property instead of a method in 6.0.

File: HelloDotNet/HelloDotNet.csproj [hunk 4]
```diff
@@ -5,7 +5,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LaunchDarkly.ServerSdk" Version="5.*" />
-    <PackageReference Include="Common.Logging" Version="3.*" />
+    <PackageReference Include="LaunchDarkly.ServerSdk" Version="6.0.0-rc.1" />
```
Comment (eli-darkly) [inline HelloDotNet/HelloDotNet.csproj]: This exact-version dependency will be changed to `6.*` after the GA release, so the demo will always pick up the latest GA version.

**Review by bwoskow-ld — APPROVED** [review]

**Review by gwhelanLD — APPROVED** [review]

**Pull request merged**